maskrcnn_benchmark/structures/segmentation_mask.py

```diff
@@ -169,46 +226,46 @@ def transpose(self, method):
             )
 
         flipped = []
-        for polygon in self.polygons:
-            flipped.append(polygon.transpose(method))
+        for mask in self.masks:
+            flipped.append(mask.transpose(method))
         return SegmentationMask(flipped, size=self.size, mode=self.mode)
 
     def crop(self, box):
-        w, h = box[2] - box[0], box[3] - box[1]
+        w, h = box[2] - box[0] + 1, box[3] - box[1] + 1
```

Comment:
Same comment as before about the +1, can we keep it as it was and add a comment instead?

Reply:
I think +1 is the correct equation, because you didn't use the `w, h` in the class `Polygons`, the performance is actually unrelated to that.

Reply:
I think a good exercise here is to try to make in such a way that [this part of the code in Detectron](https://github.com/facebookresearch/Detectron/blob/master/detectron/utils/segms.py#L110-L121) is equivalent to [this part in maskrcnn-benchmark](https://github.com/facebookresearch/maskrcnn-benchmark/blob/master/maskrcnn_benchmark/modeling/roi_heads/mask_head/loss.py#L38-L39).
I think that for them to be equivalent, we should not have the +1, but I might be wrong here.

Reply:
OK. I think for the resize operation in polygons, there should be no `+1`. But the crop operation should have `+1`.